src/redux/books/books.js

```diff
@@ -0,0 +1,19 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+const reducer = (book = [], action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...book, action.book];
+    case REMOVE_BOOK:
+      return [...book].filter((book) => book.id !== action.id);
+    default:
+      return book;
+  }
+};
+
+export default reducer;
+
+export const createBook = (book) => ({ type: ADD_BOOK, book });
+
```

Comment:
- [ ] Hello @mwafrika Nice work, you are halfway true there is just a small thing that you missed instead of exporting each const you need to make it as the duck pattern.

![Capture web_30-3-2022_131754_github com](https://user-images.githubusercontent.com/61562626/160843778-49efef5d-0931-4821-bf98-e49b9207bd0b.jpeg)